_layouts/conference.html

```diff
@@ -0,0 +1,84 @@
+---
+layout: base
+---
+
+<div class="about-page grid-wrapper conference">
+  <div class="grid__item width-12-12">
+    <h1>Wild<strong>Fly</strong> Mini Conference</h1>
+    <h2 class="page-subtitle">A dedicated conference for the community with topics around Wild<strong>Fly</strong>.
+    </h2>
+  </div>
+
+  <div class="grid__item width-12-12 sec-title-red">
+    <h2>What you need to know</h2>
+    <dl class="c-when-and-where">
+      <dt>Date</dt>
+      <dd id="c-date" data-date="2024-03-06"><a href="{{site.baseurl}}/assets/data/wildfly-mini-conference.ics" title="Click here to download an ICS file for the conference" type="text/calendar" download>Wednesday, March 6, 2024</a></dd>
+      <dt>Time</dt>
+      <dd>14:00 - 18:00 UTC</dd>
+      <dt>Location</dt>
+      <dd>
+        Virtually on <a href="https://www.youtube.com/live/_8g1rZ80u2A" target="_blank">YouTube Live</a>
+      </dd>
+    </dl>
+    <p>Registration is not required, but you can let us know that you <a href="https://forms.gle/zK6apuNk6rotXrkq5" target="_blank">plan to attend</a>. Click on the date to add the event to your calendar. You can also use the "Notify Me" button on YouTube if you would like to be notified about the conference.</p>
```

Comment:
Just a very minor comment, might be good to move "You can also use the "Notify Me"..." button to a new line just to make it stand out a bit more.

Reply:
Like this: https://hpehl.github.io/wildfly.org/conference/

Reply:
Looks good, thanks!